src/Microsoft.FeatureManagement/TelemetryConfiguration.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Generic;
+
+namespace Microsoft.FeatureManagement
+{
+    /// <summary>
+    /// Defines telemetry related configuration settings available for features.
+    /// </summary>
+    public class TelemetryConfiguration
+    {
+        /// <summary>
+        /// A flag to enable or disable sending telemetry events to the registered <see cref="Telemetry.ITelemetryPublisher"/>.
```

Comment:
I would like to drop the word "events". I suggest "publishing/sending telemetry".

Reply:
Why would you like to drop the word events. This flag controls whether `EvaluationEvent` type is pushed to registered telemetry publishers. We already use event in what seems the most important piece of the puzzle.

Reply:
Do you think it would be better to instead go in the other direction and be more specific. Such as "enable or disable sending <cref="EvaluationEvent"/> events to registered <cref="ITelemetryPublisher"/>s.

Reply:
> Do you think it would be better to instead go in the other direction and be more specific. Such as "enable or disable sending <cref="EvaluationEvent"/> events to registered <cref="ITelemetryPublisher"/>s.

Sounds good. This is much better.